flake.nix

```diff
@@ -0,0 +1,56 @@
+{
+  description = "Rust development environment";
+
```

Comment:
did you mean to check in this file?

Reply:
Nope, sorry about that. Just playing around with NixOS. Will remove.